### ISSUE TITLE
Turn sources into <ul>

### ISSUE THREAD

**Issue by cweber** [sunlightlabs/mptindicators#80]

**Comment by jcarbaugh** [comment]
I don't think this will work. The text isn't standardized enough to split and recombine into a list. I have a basic attempt, in that commit, but it doesn't work. I recommend just closing this.

**Comment by cweber** [comment]
Ok. I'll deal.

Caitlin Weber
Creative Director | Sunlight Foundation http://www.sunlightfoundation.com/
@CaitlinWeber http://twitter.com/caitlinweber
[image: Sunlight Foundation] http://sunlightfoundation.com/ [image:
Sunlight Foundation on Twitter] http://twitter.com/sunfoundation[image:
Sunlight Foundation on YouTube] http://www.youtube.com/sunlightfoundation

**Comment by jcarbaugh** [comment]
@handlers I'm assigning this to you now. It might be worth taking a look at if you have absolutely NOTHING else to do, but it seems like it's too much of a hassle due to the formatting of the data.
